twinkle.js

```diff
@@ -163,7 +163,7 @@ Twinkle.defaultConfig.friendly = {
 	// Talkback
 	markTalkbackAsMinor: true,
 	insertTalkbackSignature: true,  // always sign talkback templates
-	talkbackHeading: "Talkback",
+	talkbackHeading: "New message",
```

Comment:
```suggestion
	talkbackHeading: "New message from " + mw.config.get('wgUserName'),
```
See discussion in #513

Reply:
I don't know how to accept change, but I agree with this

Reply:
@DannyS712 	Is there not an "commit suggestion" button or something similar here in this window?

Reply:
Nope. There is "review changes" (which I think is for the overall pull request), in the top right, and when I clicked on the "start review" within this window it just adds a "1" to that button and still won't let me approve anything...

Reply:
Maybe because I am not a collaborator?

Reply:
Should be the other way around, if anything — you own this branch.

Reply:
Hmm, did you do something weird with this branch?  I notice above it says "unknown repository" for the location, which might be causing issues.

Reply:
I have no idea, sorry. Should I just open a new pull request incorporating your suggested change? Or, should this approved as is, and then another pull request with just your change can be committed? Sorry

Reply:
Okay. So should this be closed as rejected?

Reply:
Thanks again for this